### PR TITLE
Bug-1185832 Align Report and Date.

### PR DESCRIPTION
**Pull request by rahultalreja**

### CODE REVIEW

File: src/media/css/reviews.styl [hunk 1]
```diff
@@ -33,7 +33,6 @@ select[name=rating] {
         font-size: 12px;
     }
     time {
-        position: absolute;
 
         +ltr() {
```
Comment (spasovski) [inline src/media/css/reviews.styl]: I think without `absolute` the `right` and `left` properties below become meaningless.

**Comment by spasovski** [comment]
Before and after screenshots?

**Comment by tsl143** [comment]
@rahultalreja Please add before and after screen shots.

**Comment by rahultalreja** [comment]
extremely Sorry for delay!!
here goes the screen shots.
1. Before 
screenshot from 2015-10-31 14 21 15

**Comment by rahultalreja** [comment]
After
10590073_939893902716141_934394941_n

**Comment by rahultalreja** [comment]
After
after

**Comment by atiqueahmedziad** [comment]
@rahultalreja you need to update the branch with latest changes.

**Comment by ngokevin** [comment]
I think it is supposed to align horizontally, not vertically

**Comment by ngokevin** [comment]
REPORT -------------------  DATE

rather than

REPORT
DATE

**Comment by ngokevin** [comment]
let me know when this is updated and reopened, thanks!

**Comment by atiqueahmedziad** [comment]
@ngokevin but I had seen that Mark gave a design here- https://bugzilla.mozilla.org/show_bug.cgi?id=1185832#c15

**Pull request closed without merging**